pumpkin/src/world/mod.rs

```diff
@@ -259,6 +262,34 @@ impl World {
         dbg!("DONE CHUNKS", inst.elapsed());
     }
 
+    /// Sends a message to all players
+    pub fn broadcast_message(&self, content: &TextComponent) {
+        self.current_players.lock().values().for_each(|player| {
+            player.send_system_message(content.clone());
+        });
+    }
+
+    /// Gets all players
+    pub fn get_player_names(&self) -> Vec<String> {
+        self.current_players
+            .lock()
+            .values()
+            .map(|p| p.gameprofile.name.clone())
+            .collect::<Vec<_>>()
+    }
+
+    pub fn get_nearest_player_name(&self, target: &Vector3<f64>) -> Option<String> {
```

Comment:
I understand this PR is not intended to give us the functionality for everything, but this should not be `get_nearest_player_name`, but 
```rust
let Some(name) = get_nearest_player().map(Player::get_name) else {
    return Err(...)
 }
``` 
or something. 

for a single `get_nearest_...` it works, but implementing this functionality for EVERYTHING you can get from a "nearest player" would be terrifying